script_library/deployment-actions-openstack.sh

```diff
@@ -81,6 +81,7 @@ function clean_airship(){
     fi
 }
 function teardown(){
+    gather_logs
```

Comment:
I guess we do not need it always, not for local setups usually. So maybe some variable?

Reply:
good point